### PR TITLE
Parse vowels

### PR DESCRIPTION
**Pull request by jleeothon**

### CODE REVIEW

File: src/VowelLang.js [hunk 1]
```diff
@@ -19,10 +24,33 @@ export default class VowelLang {
   }
 
   /**
-   * @param {string} text A string like "open front unrounded".
+   * @param {string} text A string like "open+front+unrounded".
+   * @param {string} symbol
    * @returns {Vowel} The corresponding vowel.
-   * @example textToVowel("open front unrounded") # new Vowel(1, 1, false)
+   * @example textToVowel("open+front+unrounded") # new Vowel(1, 1, false)
```
Comment (lwille) [inline src/VowelLang.js]: adjust example to include symbol

File: src/VowelLang.js [hunk 2]
```diff
@@ -19,10 +24,33 @@ export default class VowelLang {
...
+      unrecognizedProperties.roundness = roundness
+    }
+    if (!_.isEmpty(unrecognizedProperties)) {
+      const kvPairs = _.map(unrecognizedProperties, (v) => `"${v}"`)
+      throw new TypeError(`Unrecognized ${kvPairs.join(', ')}`)
+    }
+    const o = Openness[openness]
+    const f = Frontness[frontness]
+    const r = Roundness[roundness]
+    return new Vowel(o, f, r, symbol)
```
Comment (lwille) [inline src/VowelLang.js]: suggestion: if `symbol` is not given, use `VowelList::findDefaultVowel` to try and get the default IPA symbol for the given articulation props.

File: src/VowelLang.js (hunk 1 above)
Comment (lwille) [inline src/VowelLang.js]: adjust example to include (optional) symbol

File: src/VowelLang.js (hunk 2 above)
Comment (jleeothon) [reply src/VowelLang.js]: Good one

**Pull request merged**